devsetup/Makefile

```diff
@@ -98,21 +101,25 @@ crc_attach_default_interface_cleanup: ## Detach default libvirt network from CRC
 ##@ EDPM
 
 .PHONY: edpm_baremetal
-edpm_baremetal: export PROVISIONING_INTERFACE=${BM_PROVISIONING_INTERFACE}
+edpm_baremetal: export EDPM_BMH_NAMESPACE=${BMH_NAMESPACE}
+edpm_baremetal: export EDPM_PROVISIONING_INTERFACE=${BM_PROVISIONING_INTERFACE}
+edpm_baremetal: export EDPM_CTLPLANE_INTERFACE=${BM_CTLPLANE_INTERFACE}
 edpm_baremetal: export OPERATOR_NAME=openstack
+edpm_baremetal: export EDPM_ANSIBLE_SECRET=${DATAPLANE_ANSIBLE_SECRET}
 edpm_baremetal: export EDPM_ANSIBLE_USER=${DATAPLANE_ANSIBLE_USER}
 edpm_baremetal: export BMAAS_NETWORK_NAME=${BM_NETWORK_NAME}
 edpm_baremetal: export BMAAS_NETWORK_IPADDRESS=${BM_NETWORK_IPADDRESS}
-edpm_baremetal: export EDPM_DEPLOY_STRATEGY_DEPLOY=${DATAPLANE_DEPLOY_STRATEGY_DEPLOY}
 edpm_baremetal: export BMAAS_INSTANCE_NAME_PREFIX=${BM_INSTANCE_NAME_PREFIX}
 edpm_baremetal: export EDPM_CHRONY_NTP_SERVER=${DATAPLANE_CHRONY_NTP_SERVER}
 edpm_baremetal: export EDPM_SSHD_ALLOWED_RANGES=${DATAPLANE_SSHD_ALLOWED_RANGES}
 edpm_baremetal: export EDPM_ROOT_PASSWORD_SECRET=${BM_ROOT_PASSWORD_SECRET}
 edpm_baremetal: export BMAAS_NODE_COUNT=${BM_NODE_COUNT}
 edpm_baremetal: export NODE_COUNT=${BM_NODE_COUNT}
```

Comment:
We can remove the NODE_COUNT from here I think.

Reply:
NODE_COUNT is used by vbm-setup.sh and edpm-compute-baremetal.sh.

Reply:
Yeah, we've now 3 vars for NODE_COUNT

Reply:
I know its not great, I'd like to clean this up later